### PR TITLE
Challenge completed

### PR DESCRIPTION
**Pull request by Bondesvick**
please review my solution and give me feedback

### CODE REVIEW

File: src/binary-reversal/index.js [hunk 1]
```diff
@@ -3,6 +3,14 @@
  *
  *  * @param {string} value
  */
-function binaryReversal(value) {}
+function binaryReversal(value) {
+    let reversed = Number(value)
```
Comment (bondz) [inline src/binary-reversal/index.js:7]: Nice.

This can be `const` though

File: src/list-sorting/index.js [hunk 2]
```diff
@@ -1,3 +1,21 @@
-function listSorting(needle, haystack) {}
+function listSorting(needle, haystack) {
+    if (typeof haystack[0] != "object"){
```
Comment (bondz) [inline src/list-sorting/index.js:2]: Prefer `Array.isArray` when checking if an object is an array.

File: src/list-sorting/index.js [hunk 3]
```diff
@@ -1,3 +1,21 @@
...
+
+    haystack.forEach((element, index, array) => {
+      if (element.includes(needle)) {
+        row = index
+        column = element.lastIndexOf(needle)
+        match = true
+      }
+    })
+
+    return !match ? -1 : [row, column]
```
Comment (bondz) [inline src/list-sorting/index.js:18]: A reversed condition would be easier to read. Easy to skip over the `!` sometimes.

`match ? [row, column] : -1`

File: src/binary-reversal/index.js (hunk 1 above)
Comment (Bondesvick) [reply src/binary-reversal/index.js:7]: Alright, noted

File: src/list-sorting/index.js (hunk 2 above)
Comment (Bondesvick) [reply src/list-sorting/index.js:2]: Noted

File: src/list-sorting/index.js (hunk 3 above)
Comment (Bondesvick) [reply src/list-sorting/index.js:18]: Noted

**Pull request merged**